syz-ci/jobs.go

```diff
@@ -421,6 +421,7 @@ func (jp *JobProcessor) bisect(job *Job, mgrcfg *mgrconfig.Config) error {
 			Repo:           mgr.mgrcfg.Repo,
 			Branch:         mgr.mgrcfg.Branch,
 			Commit:         req.KernelCommit,
+			CommitTitle:    req.KernelCommitTitle,
```

Comment:
tools/syz-bisect/bisect.go also fills this config.
I think we need to extend syz-bisect to support the title as well.

Reply:
looking into this.